src/game/client/swarm/vgui/nb_mission_summary.cpp

```diff
@@ -139,6 +141,26 @@ void CNB_Mission_Summary::OnThink()
 		m_pMissionLabel->SetText(pMap->m_szMissionTitle);
 	}
 
+	int nSpeedrunTime = CAlienSwarm::GetSpeedrunTime();
+	wchar_t wszSRTime[ 128 ];
+
+	if ( nSpeedrunTime <= 0 )
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), L"-" );
+	else
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), g_pVGuiLocalize->FindSafe( "#nb_speedruntime_format" ), nSpeedrunTime / 60, nSpeedrunTime % 60 / 10, nSpeedrunTime % 10 );
+
+	m_pSpeedrunTimeLabel->SetText( wszSRTime );
```

Comment:
should we also show par time?

Reply:
is it accurate? how is it calculated?

Reply:
https://github.com/ReactiveDrop/reactivedrop_public_src/blob/9760978fd7bf3a982b9373d9a6bebf53ab0da8ee/src/game/client/swarm/vgui/experience_stat_line.cpp#L44

Reply:
meh